c/secp256k1_keccak256_lock_tron.h

```diff
@@ -0,0 +1,47 @@
+ /* The script perform secp256k1_keccak256_sighash_all verification. */
+#define BLAKE2B_BLOCK_SIZE 32
+#define BLAKE160_SIZE 20
+#define PUBKEY_SIZE 65  // ETH address uncompress pub key 
+#define TEMP_SIZE 32768
+#define RECID_INDEX 64
+/* 32 KB */
+#define MAX_WITNESS_SIZE 32768
+#define SCRIPT_SIZE 32768
+#define SIGNATURE_SIZE 65
+
+#if (MAX_WITNESS_SIZE > TEMP_SIZE) || (SCRIPT_SIZE > TEMP_SIZE)
+#error "Temp buffer is not big enough!"
+#endif
+
+
+/*
+ * Arguments:
```

Comment:
This function no longer reads script args and transaction witness, this comment block is obsoleted and misleading.

Reply:
The following functions have the same issue:

* `verify_secp256k1_keccak_tron_sighash_all`
* `verify_secp256k1_keccak_eth_sighash_all`